src/vorta/views/main_window.py

```diff
@@ -125,7 +113,17 @@ def profile_delete_action(self):
 
             to_delete.delete_instance(recursive=True)
             self.profileSelector.removeItem(self.profileSelector.currentIndex())
-            self.profile_select_action(1)
+            self.profile_select_action(0)
+
+    def profile_add_action(self):
+        window = AddProfileWindow()
+        window.setParent(self, QtCore.Qt.Sheet)
+        window.show()
+        if window.exec_() and window.edited_profile:
+            self.profileSelector.addItem(window.edited_profile.name, window.edited_profile.id)
+            self.profileSelector.setCurrentIndex(self.profileSelector.count() - 1)
+        else:
+            self.profileSelector.setCurrentIndex(self.profileSelector.currentIndex())
```

Comment:
When cancelling the add profile window, we now keep to the current profile selected instead of selecting the default profile.